assets/js/payment-method-extensions/payment-methods/stripe/payment-request/index.js

```diff
@@ -16,30 +16,47 @@ const ApplePayPreview = () => <img src={ applePayImage } alt="" />;
 const canPayStripePromise = loadStripe();
 const componentStripePromise = loadStripe();
 
+let apiInitialized = false,
```

Comment:
It's convention to use the `is` prefix on variables holding boolean type values (especially when they are in conditional expressions) as it reads more naturally. I think `isInitialized` is sufficient because the scope is already limited to this file but if you want to include api in there then `isApiInitialized` or even `isStripeInitialized` if you want to be super specific.